backend-rust/src/indexer.rs

```diff
@@ -2269,8 +2358,23 @@ impl PreparedBakerEvent {
             PreparedBakerEvent::RemoveDelegation {
                 delegator_id,
             } => {
+                // Update total pool stake of old pool (if not the passive pool).
+                sqlx::query!(
+                    "UPDATE bakers
+                     SET pool_total_staked = pool_total_staked - accounts.delegated_stake
```

Comment:
Also update the `pool_delegation_count` value.

Reply:
Yes, great catch!